Error building posix_sitl_default

Hi, 

I'm getting an error while building PX4 for SITL simulation with Gazebo. I installed the pre-requisites as stated in [this page](http://dev.px4.io/simulation-gazebo.html) and ran `make posix_sitl_default gazebo`. I got the following error:
```
--   Found protobuf, version 3.0.2
Gazebo version: 7.3
CMake Error at CMakeLists.txt:80 (message):
  protobuf version: 3.0.2 not compatible, must be >= 2.5.0 and < 3.0.0


-- Configuring incomplete, errors occurred!
```

So it appears that the protobuf version I have is too new, therefore I downloaded the newest version below v3.0.0 (as the error log is suggesting) which is v2.6.1 and I installed it. Then I proceeded to re-run the build as follows:

```
make posix_sitl_default clean
make posix_sitl_default gazebo
```

I get a new error which is very long, but I'll only display the beginning since it says where it does come from:

```
In file included from SensorImu.pb.cc:5:
In file included from ./SensorImu.pb.h:27:
/usr/local/Cellar/gazebo7/7.3.1_4/include/gazebo-7/gazebo/msgs/quaternion.pb.h:12:2: error: This file was generated by a newer version of protoc which is
#error This file was generated by a newer version of protoc which is
 ^
/usr/local/Cellar/gazebo7/7.3.1_4/include/gazebo-7/gazebo/msgs/quaternion.pb.h:13:2: error: incompatible with your Protocol Buffer headers.  Please update
#error incompatible with your Protocol Buffer headers.  Please update
 ^
/usr/local/Cellar/gazebo7/7.3.1_4/include/gazebo-7/gazebo/msgs/quaternion.pb.h:14:2: error: your headers.
#error your headers.
 ^
```
So now it seems like some headers that are used by protobuf has been generated by a newer version, and thus it is not able to recognize them. The problem is that there is no other version in [the releases of protobuf page](https://github.com/google/protobuf/releases?after=v3.0.0-alpha-1) so I don't know where the problem comes from.

I'm building all of this in a Mac with gazebo7 installed today and with the latest version of the master firmware.


P.S: If I change the CMakeLists.txt in `Tools/sitl_gazebo` and allow for having protobuf versions above 3.0.0 the simulation executes but without the world.

--- a/starting-installing-mac.md
+++ b/starting-installing-mac.md
@@ -23,7 +23,14 @@ brew update
 brew install git bash-completion genromfs kconfig-frontends gcc-arm-none-eabi
 brew install astyle cmake ninja
 # simulation tools
-brew install ant graphviz sdformat3 protobuf eigen opencv
+brew install ant graphviz sdformat3 eigen opencv
+```
+
+We need to get an older version of protobuf (`< 3.0.0`).
+
+```sh
+brew tap homebrew/versions
+brew install homebrew/versions/protobuf260
 ```
 
 Then install the required python packages:

